aspnetcore/index.yml

```diff
@@ -0,0 +1,455 @@
+      links:
+        - url: getting-started/index.md
+          itemType: get-started
+          text: "Create an ASP.NET Core app on any platform in 5 minutes"
+        - url: tutorials/build-your-first-blazor-app.md
+          itemType: get-started
+          text: "Create a client-side web UI in C# with ASP.NET Core Blazor"
+    # Card
+    - title: Create a web API
+      summary: "Create your first web API in ASP.NET Core"
+      links:
+        - url: tutorials/first-web-api.md
+          itemType: get-started
+          text: "Create a RESTful web API with ASP.NET Core"
+        - url: tutorials/grpc/grpc-start.md
+          itemType: get-started
+          text: "Create a gRPC service with ASP.NET Core"
+    # Card
+    - title: Create a real-time web app
+      summary: "Create your first real-time web app in ASP.NET Core"
+      links:
+        - url: tutorials/signalr.md
+          itemType: get-started
+          text: "Create a real-time web app with ASP.NET Core SignalR"
+    # Card
+    - title: Create a data-driven web app
+      summary: "Create your first data-driven web apps"
+      links:
+        - url: tutorials/first-mvc-app.md
+          itemType: get-started
+          text: "Create a web app with ASP.NET Core MVC & SQL Server LocalDB"
+        - url: data/ef-rp/intro.md
+          itemType: get-started
+          text: "Razor Pages & Entity Framework Core"
+        - url: data/ef-mvc.md
+          itemType: get-started
+          text: "ASP.NET Core MVC with Entity Framework Core"
+        - url: https://docs.microsoft.com/en-us/learn/modules/build-web-api-net-core/
+          itemType: get-started
+          text: "Create a page-focused web UI that consumes a RESTful web API"
+
+# conceptualContent section (optional)
+conceptualContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  title: Develop ASP.NET Core apps # < 60 chars (optional)
+  summary: "Choose interactive web apps, web API, MVC-patterned apps, real-time apps, and more"
+  items:
+    # Card
+    - title: "Interactive client-side Blazor apps"
+      summary: Develop with reusable UI components that can take advantage of WebAssembly for near-native performance
+      links:
+        - url: blazor/index.md
+          itemType: overview
+          text: "Overview"
+        - url: blazor/get-started.md
+          itemType: get-started
+          text: "Create your first Blazor app"
+        - url: tutorials/build-your-first-blazor-app.md
+          itemType: get-started
+          text: "Build your first Blazor app with reusable components"
+        - url: blazor/hosting-models.md
+          itemType: concept
+          text: "Blazor hosting models"
+    # Card
+    - title: RESTful web API apps
+      summary: Develop RESTful HTTP services with ASP.NET Core web API
+      links:
+        - url: web-api/index.md
+          itemType: overview
```

Comment:
This is not an overview. Move this 2nd.

Reply:
@Rick-Anderson True, thanks. (re: web-api/index.md) It is not an overview.  We call it an overview currently in the TOC, and as far as I know it is the only doc Microsoft has that provides a brief overview description of ASP.NET Core web API.  Where else could we send developers for any overview information?  I think this is the best we have at the moment if folk want an explanation of what it is before they commit to working through a tutorial.

Reply:
Moving it to item 2 as you suggested after looking through both tutorials again.  That is the better fit.  Thanks.